examples/gly12_hf_energy.in

```diff
@@ -1,91 +0,0 @@
- HF BASIS=cc-pvDZ CUTOFF=1.0d-10 DENSERMS=1.0d-8 ENERGY
```

Comment:
File name: Add also basis set. `gly12_hf_cc-pvdz_energy.in`.

Reply:
We no longer have an examples folder in QUICK. I plan to maintain a collection of examples in a separate repository.